internal/mode/static/nginx/config/execute_test.go

```diff
@@ -3,27 +3,25 @@ package config
 import (
 	"testing"
 
+	. "github.com/onsi/gomega"
+
 	"github.com/nginxinc/nginx-kubernetes-gateway/internal/mode/static/nginx/config/http"
 )
 
 func TestExecute(t *testing.T) {
 	defer func() {
-		if r := recover(); r != nil {
-			t.Errorf("execute() panicked with %v", r)
-		}
+		g := NewWithT(t)
+		g.Expect(recover()).Should(BeNil())
 	}()
-
+	g := NewWithT(t)
```

Comment:
Could we just define this at the top of the test and reuse it in the defer func, rather than defining it twice?